### PR TITLE
fix?

### PR DESCRIPTION
**Pull request by grahamegrieve**

### CODE REVIEW

File: rX/snapshot-generation/nl-core-NursingIntervention-output.json [hunk 1]
```diff
@@ -1315,9 +1315,10 @@
...
-      "definition" : "A reference from one resource to another.",
-      "comment" : "Each occurrence of the zib HealthProfessional is normally represented by _two_ FHIR resources: a PractitionerRole resource (instance of [nl-core-HealthProfessional-PractitionerRole](http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole)) and a Practitioner resource (instance of [nl-core-HealthProfessional-Practitioner](http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-Practitioner)). The Practitioner resource is referenced from the PractitionerRole instance. For this reason, sending systems should fill the reference to the PractitionerRole instance here, and not the Practitioner resource. Receiving systems can then retrieve the reference to the Practitioner resource from that PractitionerRole instance.\r\n\r\nIn rare circumstances, there is only a Practitioner instance, in which case it is that instance which will be referenced here. However, since this should be the exception, the nl-core-HealthProfessional-Practitioner profile is not explicitly mentioned as a target profile.",
+      "short" : "Requester",
```
Comment (pieter-edelman-nictiz) [inline rX/snapshot-generation/nl-core-NursingIntervention-output.json:1318]: `.comment` now reverts to the base definition from FHIR. This is not the desired behavior, it should be the value from the data type profile used here.

On the other hand, `.short`, `.definition` and `.alias` now use the correct values (they come from the base profile and are not overridden by anything).

File: rX/snapshot-generation/nl-core-NursingIntervention-output.json [hunk 2]
```diff
@@ -1341,12 +1342,9 @@
       "isModifier" : false,
       "isSummary" : false,
       "mapping" : [{
-        "identity" : "rim",
-        "map" : "n/a"
-      },
-      {
-        "identity" : "rim",
-        "map" : "The target of a resource reference is a RIM entry point (Act, Role, or Entity)"
+        "identity" : "zib-nursingintervention-v3.2-2020EN",
```
Comment (pieter-edelman-nictiz) [inline rX/snapshot-generation/nl-core-NursingIntervention-output.json:1345]: Adding this mapping is the desired behavior.

File: rX/snapshot-generation/nl-core-NursingIntervention-output.json [hunk 3]
```diff
@@ -1422,8 +1420,8 @@
       "id" : "CarePlan.contributor:requester.extension:contributorIsRequester",
       "path" : "CarePlan.contributor.extension",
       "sliceName" : "contributorIsRequester",
-      "short" : "Optional Extensions Element",
-      "definition" : "Optional Extension Element - found in all resources.",
+      "short" : "Extension",
```
Comment (pieter-edelman-nictiz) [inline rX/snapshot-generation/nl-core-NursingIntervention-output.json:1423]: The extension is not included in the test case. Should this be added as well?

File: rX/snapshot-generation/nl-core-NursingIntervention-output.json [hunk 4]
```diff
@@ -2194,7 +2184,6 @@
         "code" : "Extension",
         "profile" : ["http://nictiz.nl/fhir/StructureDefinition/ext-CarePlan-MaterialUsed"]
       }],
-      "condition" : ["ele-1"],
```
Comment (pieter-edelman-nictiz) [inline rX/snapshot-generation/nl-core-NursingIntervention-output.json:2197]: Not sure why this `condition` is removed. Shouldn't it always be present?

File: rX/snapshot-generation/nl-core-NursingIntervention-output.json [hunk 5]
```diff
@@ -2318,7 +2300,6 @@
         }],
         "code" : "http://hl7.org/fhirpath/System.String"
       }],
-      "fixedUri" : "http://nictiz.nl/fhir/StructureDefinition/ext-CarePlan-MaterialUsed",
```
Comment (pieter-edelman-nictiz) [inline rX/snapshot-generation/nl-core-NursingIntervention-output.json:2321]: Not sure if this is due to the fact that the extension is not included in the test case, but I think it should not be removed?

File: rX/snapshot-generation/nl-core-NursingIntervention-output.json [hunk 6]
```diff
@@ -4100,10 +4081,11 @@
...
+      "short" : "HealthcareProvider",
+      "definition" : "The health professional carrying out the nursing procedure.",
+      "comment" : "A performer MAY also be a participant in the care plan.",
```
Comment (pieter-edelman-nictiz) [inline rX/snapshot-generation/nl-core-NursingIntervention-output.json:4086]: Like above:

.comment now reverts to the base definition from FHIR. This is not the desired behavior, it should be the value from the data type profile used here.

On the other hand, .short, .definition and .alias now use the correct values (they come from the base profile and are not overridden by anything).

File: rX/snapshot-generation/nl-core-NursingIntervention-output.json [hunk 7]
```diff
@@ -4127,12 +4109,21 @@
       "isModifier" : false,
       "isSummary" : false,
       "mapping" : [{
-        "identity" : "rim",
-        "map" : "n/a"
+        "identity" : "zib-nursingintervention-v3.2-2020EN",
```
Comment (pieter-edelman-nictiz) [inline rX/snapshot-generation/nl-core-NursingIntervention-output.json:4112]: Inheriting this mapping key is the desired behavior.

**Review by pieter-edelman-nictiz — CHANGES_REQUESTED** [review]

**Comment by grahamegrieve** [comment]
Closing this - the test itself was set up wrong. See next PR

**Pull request closed without merging**